src/pages/documentation/getting-started/get-the-sdk.md

```diff
@@ -0,0 +1,108 @@
+import AddDependenciesIosACP from './tabs/get-the-sdk/add-dependencies/ios-acp.md'
+import AddDependenciesIosAEP from './tabs/get-the-sdk/add-dependencies/ios-aep.md'
+import AddDependenciesAndroid from './tabs/get-the-sdk/add-dependencies/android.md'
+import AddInitializationCodeIosACP from './tabs/get-the-sdk/add-initialization-code/ios-acp.md'
+import AddInitializationCodeIosAEP from './tabs/get-the-sdk/add-initialization-code/ios-aep.md'
+import AddInitializationCodeAndroid from './tabs/get-the-sdk/add-initialization-code/android.md'
+import GetTheSdkIosACP from './tabs/get-the-sdk/get-the-sdk/ios-acp.md'
+import GetTheSdkIosAEP from './tabs/get-the-sdk/get-the-sdk/ios-aep.md'
+import GetTheSdkAndroid from './tabs/get-the-sdk/get-the-sdk/android.md'
+
+# Get the Experience Platform SDK
+
+The Adobe Experience Platform SDK is available for Apple iOS (includes iOS, iPadOS, and tvOS) via [Cocoapods](https://cocoapods.org/) and for Google Android via [Gradle](https://gradle.org).
+
+Follow the directions below to include the SDK into your mobile application.
+
+<InlineAlert variant="info" slots="text"/>
+
+For iOS and Android projects, the recommended approach for integrating the SDK is to use supported dependency and package managers as listed for each platform such as Maven and Cocoapods. SDKs are also available for download on [GitHub](https://github.com/Adobe-Marketing-Cloud/acp-sdks/).
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="3"/>
+
+Android
+
+<GetTheSdkAndroid/>
+
+iOS (AEP 3.x)
+
+<GetTheSdkIosAEP/>
+
+iOS (ACP 2.x)
```

Comment:
for phase 1 of the migration can you also include React Native and Flutter (this applies for all pages)

Reply:
@emdobrin - This page was done before that requirement was re-added. I'm unsure that, with the limited time frame and my bandwidth, if I'll be able to re-implement those sections in time.